src/sentry/api/endpoints/organization_has_mobile_app_events.py

```diff
@@ -43,8 +43,8 @@ def _get(self, request, organization):
             # TODO: update discover query for null browser and client_os_name is android or ios
             result = discover.query(
                 query=query,
-                orderby="browser.name",
-                selected_columns=["browser.name", "project", "id", "client_os.name"],
+                orderby="-timestamp",
+                selected_columns=["browser.name", "client_os.name", "timestamp"],
                 limit=1,
                 params={
                     "start": timezone.now() - timedelta(days=1),
```

Comment:
Just out of curiosity is this only supposed to tell you if one event with a non null browser name field was sent in the last day? We are intentionally excluding results earlier than this window?

Reply:
@lynnagara It's supposed to tell us if we have an event that matches a set of 4 browsers. I just picked the last 24 hours cause we do that for the global SDK updates; it's somewhat arbitrary.

Reply:
Fair enough